config/packages/framework.yaml

```diff
@@ -16,3 +19,9 @@ framework:
 
 parameters:
     env(CLOUDINARY_CONFIG_FILE): '%kernel.project_dir%/config/cloudinary.yml'
+
+when@test:
+    framework:
+        test: true
+        session:
+            storage_factory_id: session.storage.factory.mock_file
```

Comment:
Why sessions, we don't have sessions and no UI?

Reply:
Flex defaults